src/compose/volume.ts

```diff
@@ -1,16 +1,15 @@
 import * as Docker from 'dockerode';
-import assign = require('lodash/assign');
-import isEqual = require('lodash/isEqual');
-import omitBy = require('lodash/omitBy');
```

Comment:
The usual reason to import lodash as `lodash/$x` is in order to tree shake and drop unused code for a smaller webpack output, using straight `lodash` requires importing the entirety of lodash which is quite large. I'm assuming that's the reason the supervisor had these as individual imports rather than importing everything

Reply:
Shouldn't tree shake work either way? If I do

```
import module from 'mod';

mod.somefunc()
```

Shouldn't tree shaking get rid of everything except for `somefunc` and functions it requires?

Reply:
> Btw just so you know there's no particular benefit atm to using import = require vs import * because typescript will transpile them both to the same thing

@Page- I know, the only reason I'm doing this is because I'm trying to generate a dependency tree diagram and these `require` imports show as `unresolved module` for some reason I haven't been able to figure out. Is a bonus that we get consistent code :)

See this thread for more context https://www.flowdock.com/app/rulemotion/r-supervisor/threads/ophCVmMO6NIY5axDZ4EkKe2focX

Reply:
Tree-shaking only works on a per-export method for es-modules because it requires the semantic restrictions of the es-module syntax in order to prove the tree-shaking is safe, and as lodash isn't actually an es-module (ts just emulates it) it needs to include the entire module

Reply:
@Page- I found this article https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/ and it looks like lodash4 is exported as true esModule. 

I was also looking at our webpack config and I don't think we do any tree-shaking, only minimization. Maybe we could enable it?